### ISSUE TITLE
Hermes Debugging: Chrome dev tools does not start when pressing the "j" key

### ISSUE THREAD

**Issue by SDP190** [expo/expo-cli#4432]
### Summary

Hi, 

My app.json is configured to use the hermes jsEngine.

After running **expo-start --localhost**, opening the app on android with Expo Go, when pressing "j" I am getting a log which tells that the dev tools are opening in the browser. however, the browser actually does not get started! no window appears! It did seem like it tries to open Chrome, but it closes immediately.

In order to investigate it, I was able to capture a powershell script which is invoked by the expo cli in order to start the chrome process, it was the following which is being called:

```
Start -Wait "`"chrome`"" -ArgumentList "`"--app=https://chrome-devtools-frontend.appspot.com/serve_rev/@e3cd97fc771b893b7fd1879196d1215b622c2bed/inspector.html?panel=sources&v8only=true&ws=%5B%3A%3A1%5D%3A19000%2Finspector%2Fdebug%3Fdevice%3D1%26page%3D-1`"","`"--allow-running-insecure-content`"","`"--user-data-dir=C:\Users\sprizant\AppData\Local\Temp\expo-inspector`"","`"--no-first-run`"","`"--no-startup-window`"","`"--no-default-browser-check`""
```

I have tried the above manually it behaved the same, chrome was not opening (at least not visible). However I ran the script without the **--no-startup-window** argument, and then the devtools did open successfully!

### Environment

 expo-env-info 1.0.3 environment info:
    System:
      OS: Windows 10 10.0.19044
    Binaries:
      Node: 16.15.1 - C:\Program Files\nodejs\node.EXE       
      Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.14.11 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-212.5712.43.2112.8609683
    npmPackages:
      babel-preset-expo: ~9.1.0 => 9.1.0 
      expo: ^45.0.0 => 45.0.5 
      react: 17.0.2 => 17.0.2 
      react-dom: 17.0.2 => 17.0.2 
      react-native: 0.68.2 => 0.68.2 
      react-native-web: 0.17.7 => 0.17.7 
    Expo Workflow: managed

### Please specify your device/emulator/simulator platform, model and version

AVD Emulator: Model: sdk_gphone64_x86_64

### Reproducible demo or steps to reproduce from a blank project

hg

**Comment by SDP190** [comment]
Seems like the exact same issue was mentioned by someone else in this expo forums post:
https://forums.expo.dev/t/debugging-with-hermes/64554

**Comment by yolpsoftware** [comment]
Yes, similar issue [here](https://forums.expo.dev/t/debugging-with-hermes/64554/2), but on iOS.

**Comment by SDP190** [comment]
Thanks @yolpsoftware for informing.

After digging around I can confirm, this issue has been affected by changes made to the Chrome browser.

After testing multiple times using the same environment, I can confirm: 
Chrome Version (stable): **101.0.4951.0 = Works great and starts the Chrome DevTools**
Next Chrome Version (stable): **102.0.5005.63 = Chrome Dev Tools wont open when pressing the `j` key**

As mentioned above, when tested manually using the underlying power-shell command without the "--no-startup-window" flag , then it does work well, and open's the Dev Tools.
After digging around some more, I think we can relate this behavior change to changes submitted by [this chromium commit](https://chromium.googlesource.com/chromium/src/+/4b7d53d2bc9be13b91df7f3e83ff786d04531b4b) which is effecting the "--no-startup-window" flag (was introduced as a fix for [this chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1300109)), and was merged into the  **102.0.5005.63** release according to [this release notes](https://chromium.googlesource.com/chromium/src/+log/101.0.4951.67..102.0.5005.63?pretty=fuller&n=10000)

That said, not sure why expo-cli needs to issue the "--no-startup-window" flag, just get rid of it, and all works well.

**Comment by brentvatne** [comment]
@Kudo - i pushed https://github.com/expo/expo-cli/commit/655a963929a2d4bc24b728be02ad1736c1c6151d to fix this for now because i verified this indeed resolves the issue for me locally. happy to revert / change / publish a new release when you have time to provide background information on this

**Comment by brentvatne** [comment]
the fix from 655a963929a2d4bc24b728be02ad1736c1c6151d is live in expo-cli@5.4.12
